assets/js/phoenix_live_view/view.js

```diff
@@ -828,6 +839,27 @@ export default class View {
     if(this.liveSocket.isConnected()){ this.liveSocket.reloadWithJitter(this) }
   }
 
+  redirectWithJitter() {
+    clearTimeout(this.redirectWithJitterTimer)
+    let minMs = this.redirectJitterMin
+    let maxMs = this.redirectJitterMax
+    let afterMs = Math.floor(Math.random() * (maxMs - minMs + 1)) + minMs
+    let redirectAttempts = Browser.updateLocal(this.localStorage, window.location.pathname, CONSECUTIVE_REDIRECTS, 0, count => count + 1)
+    if (redirectAttempts > this.maxRedirects) {
+      afterMs = this.redirectFailsafeJitter
+    }
+  
+    this.redirectWithJitterTimer = setTimeout(() => {
+      this.log("error", () => [`encountered ${redirectAttempts} consecutive redirects`])
+      if (redirectAttempts > this.maxRedirects) {
+        this.log("error", () => [`exceeded ${this.maxRedirects} consecutive redirects. Entering failsafe mode`])
+      }
+      if (this.isMain()) {
+        this.onRedirect({ to: this.href })
+      }
+    }, afterMs)
```

Comment:
For some reason this timeout function isn't running.

I copied the general pattern from https://github.com/phoenixframework/phoenix_live_view/blob/main/assets/js/phoenix_live_view/live_socket.js#L312-L336